ldk/javascript/src/whisper/types.ts

```diff
@@ -238,13 +240,35 @@ export type Message = WhisperComponent<WhisperComponentType.Message> & {
 };
 
 export type DropZone = WhisperComponent<WhisperComponentType.DropZone> & {
+  /**
+   * Specify what extensions the user can include. Do not include the dot.
+   *
+   * @example ['jpg','jpeg','png']
```

Comment:
Do we want to limit the file extension types that can be sent inside of the LDK?

Reply:
I don't think we'd want to create an allowlist inside the LDK itself? It could be overrode if they accessed the internal `oliveHelps` object and the author isn't required to provide an extension list. + we don't know what extensions a Loop may want to work with.